app/javascript/tests/public_health/bulk_actions/UpdateAssignedUser.test.js

```diff
@@ -0,0 +1,109 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import InfoTooltip from '../../../components/util/InfoTooltip';
+import UpdateAssignedUser from '../../../components/public_health/bulk_actions/UpdateAssignedUser';
+import { mockPatient1, mockPatient2, mockPatient6 } from '../../mocks/mockPatients';
+
+const onCloseMock = jest.fn();
+const mockToken = 'testMockTokenString12345';
+const mockAssignedUserDatalist = ['3682', '903874', '73892', '1689'];
+
+function getWrapper(patientsArray) {
+  let wrapper = shallow(<UpdateAssignedUser authenticity_token={mockToken} patients={patientsArray} close={onCloseMock} assigned_users={mockAssignedUserDatalist} />);
```

Comment:
```
return shallow(<UpdateAssignedUser authenticity_token={mockToken} patients={patientsArray} close={onCloseMock} assigned_users={mockAssignedUserDatalist} />);
```

Reply:
I think you can hardcode the `patientsArray` here as `[mockPatient1, mockPatient2]`, all the tests still pass

Reply:
you can still just return the shallow here instead of declaring then returning